### PR TITLE
Add sorting events by location

### LINKED ISSUE #60: 🌟 events that involve 2733

### PR DESCRIPTION
**Pull request by fharding1**
Closes #60. It's a different solution to the same problem, because we all thought it was cleaner than using :star:

### CODE REVIEW

File: src/utils.tsx [hunk 1]
```diff
@@ -69,17 +69,61 @@ const formatMatchId = (matchId: string): string => {
...
+              e.lat,
+              e.long
+            )
+          }
+
           return e
         })
-        .sort((a, b) => (a.distanceFromToday > b.distanceFromToday ? 1 : -1))
+        .sort((a, b) => {
+          if (a.distanceFromToday > b.distanceFromToday) {
```
Comment (calebeby) [inline src/utils.tsx]: I think this would be nicer as a ternary. Do you?

Comment (calebeby) [reply src/utils.tsx]: Actually nvm this is how prettier wants to format it:

```js

Comment (fharding1) [reply src/utils.tsx]: Not with something so long tbh. I think ternaries within ternaries are kinda gross.

Comment (fharding1) [reply src/utils.tsx]: Yeah

Comment (calebeby) [reply src/utils.tsx]: Prettier formats nested ternaries pretty nice when it is in JSX, I expected it to do that here but ¯\_(ツ)_/¯

**Pull request merged**